TS/scripts/Set Window Size.lobster

```diff
@@ -0,0 +1,7 @@
+// Set the window size to a predefined value.
+// Use case: Program sharing over common messaging software
+
+//ts_set_window_size(1920, 1080)
+ts_set_window_size(1366, 768)
```

Comment:
I don't think it makes sense to add an example script file that has these hardcoded sizes in it, as that isn't useful to anyone

Reply:
Thanks for this feedback. I did not know how to get a user input - is there a way to show a choice dialog or something?

Reply:
I don't think we do currently... but even so, inputting size in dialog is not that useful compared to just resizing the window. I suggest to remove this file from the PR and keep it for your personal use.

Reply:
Agreed! I removed the script file. Is everything ok now?